farm/evaluation/msmarco_passage_farm.py

```diff
@@ -0,0 +1,85 @@
+from farm.evaluation.msmarco_passage_official import compute_metrics_from_files
+import os
+import pandas as pd
+
+
+def msmarco_evaluation(preds_file, dev_file, qrels_file, output_file):
+    """
```

Comment:
One line describing what type of evaluation we are doing here (e.g. with link to task website)

Reply:
Also make the relation to the official script clearer. As I understood it, this is a wrapper to bring the predictions into the right shape and then run the official eval function "compute_metrics_from_files"?